README.md

```diff
@@ -77,8 +77,24 @@ To test the build, run `cargo test`.
   ```
 - **Windows**
   
-  On Windows, both `R` and `Rtools` should be available on `PATH` for tests to succeed. Ensure that `R_HOME` points to `R` home, e.g. `C:\Program Files\R\R-4.1.0`. 
-  For `x64`, append the following to the `PATH` (using `PowerShell` syntax):
+  On Windows, first, ensure that `R_HOME` points to `R` home, e.g. `C:\Program Files\R\R-4.1.0` (on an R session, this should be set by R).
+  
+  Then, also ensure `PATH` is properly configured that the following executables are available:
+  
+  * the `R` binary to build against
+  * the compiler toolchain that is used for compiling the R itself, i.e., `Rtools`
+
+  Typically, these are in the following locations:
+
+  |                  | R                         | Rtools                             |
+  | ---------------- | ------------------------- | ---------------------------------- |
+  | R <=4.1.x, 64-bit  |  `${env:R_HOME}\bin\x64`  | `${env:RTOOLS40_HOME}\mingw64\bin` |
+  | R <=4.1.x, 32-bit  |  `${env:R_HOME}\bin\i386` | `${env:RTOOLS40_HOME}\mingw32\bin` |
+  | R 4.2.x, 64-bit    |  `${env:R_HOME}\bin\x64`  | `${env:RTOOLS40_HOME}\ucrt64\bin` (this might be changed when [Rtools42] gets released) |
+  
+  [Rtools42]: https://www.r-project.org/nosvn/winutf8/ucrt3/web/rtools.html
+
+  So, for example, if the target is 64-bit R (<=4.1.x), add the following to the `PATH` (using `PowerShell` syntax). Note that this shows how to "apeend" these for minimum side effect, but, if `PATH` already contains another version of `R` or compiler toolchain, these should be prepended to overwrite the existing ones.
```

Comment:
Space in `<= 4.1.x`. I suggest modifying this paragraph and splitting it in two. First sentence should end with a colon and should be followed by the actual powershell code that shows how to modify `PATH`. After the code block, write the second sentence, but perhaps rephrase it a little bit (also there is a typo in append):

'Note that this shows how to *append paths* for minimum side effect, but if `PATH` already contains *paths to* another version of `R` *binary* or compiler toolchain, *these paths* should *instead* be prepended to *take precedence*.'.